.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: ruby
+rvm:
+- 2.2.4
+before_install: gem update --remote bundler
+install:
+- bundle install --retry=3
+script:
+- bundle exec rspec
+- bundle exec rubocop --config .rubocop.yml
+deploy:
+  provider: heroku
+  run:
+  "rake db:migrate"
```

Comment:
@jy300x2 ^ yes looks like that is causing the build to fail currently.

Can see the logs here: https://travis-ci.org/Team-Chess-Fusion/chess-fusion/builds/147090274

See if you can get the build to go green. Will probably require some codestyle changes for rubocop to be happy. Let me know if you need help. 👍